### PR TITLE
New home page

### PR DESCRIPTION
**Pull request by xfq**
Richard has created new 'home pages' for all of the lreq repositories.  For repositories without an index.html file at the top level, this was a question of refreshing the README page, and then linking to it via github.io (eg. https://w3c.github.io/afrlreq/ ).  For repos like clreq and jlreq, however, that won't work, so he created a new home.md file which we can link to as https://w3c.github.io/clreq/home and https://w3c.github.io/jlreq/home .

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/pull/356.html" title="Last updated on Apr 20, 2021, 6:00 AM UTC (3ef4a86)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/356/816bcdf...3ef4a86.html" title="Last updated on Apr 20, 2021, 6:00 AM UTC (3ef4a86)">Diff</a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,36 @@
-# Chinese Text Layout task force (clreq) | 中文排版需求
+# Chinese Language Enablement (clreq) | 中文语言支持
 
-This group allows a network of experts to share information about gaps and requirements for support of Simplified and Traditional Chinese on the Web and in eBooks. 
+This is the place to explore gaps in support for the Chinese Simplified and Traditional scripts on the Web and in eBooks, and to document requirements.
 
-本小组探讨在网页和电子书中使用简体、繁体中文时的需求以及目前差距。
+本小组旨在探讨 Web 和电子书中使用简体、繁体中文时的差距，并且记录相关需求。
```
Comment (realfish) [inline README.md]: 「探讨 Web 和电子书中使用简体、繁体中文时的差距」会不会被误解为「difference between SC and TC on the Web」？改成「探讨 Web 和电子书在简体和繁体中文支持方面的不足」会不会好些？

**Review by realfish — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xfq) [reply README.md]: 已修改，谢谢。

**Comment by r12a** [comment]
Looks good to me.

But should we create a public-i18n-chinese list for notifications and minutes? (To replace the public-i18n-cjk list.)  If so, we should point to that.

**Review by r12a — APPROVED** [review]

**Comment by xfq** [comment]
> But should we create a public-i18n-chinese list for notifications and minutes? (To replace the public-i18n-cjk list.) If so, we should point to that.

Done. Thank you.

**Pull request merged**